app/templates/components/edit-reminder.hbs

```diff
@@ -12,6 +12,15 @@
     {{textarea class="spec-edit-textarea-notes" value=model.notes}}
   </label>
   {{#link-to "reminders.reminder-card"}}
-    <button class='spec-reminder-edit--save'>Save</button>
+    <button
+      class='spec-reminder-edit--save'
+      {{action 'editReminder' model on="click"}}>
+      Save
+    </button>
   {{/link-to}}
+  <button
+    class='spec-reminder-edit--revert'
+    {{action 'revertReminder' model on="click"}}>
```

Comment:
Let's add a conditional here that only displays this button (or only makes it active - your choice) if there are dirty attributes on the model. [docs](https://guides.emberjs.com/v2.11.0/models/creating-updating-and-deleting-records/#toc_persisting-records)